src/components/form-grid/index.js

```diff
@@ -15,38 +15,12 @@ import { define } from "../../utils/core";
  */
 export class VlFormGrid extends VlGrid {
   connectedCallback() {
-    this.classList.add("vl-form-grid");
+    this.classList.add('vl-grid');
```

Comment:
Wow, wat gebeurt er hier? Is er onderzocht wat er mis was met de `vl-form-grid`? Dat zal er niet zomaar hebben gestaan. Op dit punt heeft de `vl-form-grid` geen enkele meerwaarde tegenover een normale `vl-grid`

Reply:
Revert. Unit test werkt terug.